provider/pkg/provider/provider.go

```diff
@@ -459,6 +457,17 @@ func (p *cfnProvider) Configure(ctx context.Context, req *pulumirpc.ConfigureReq
 		}
 	}
 
+	if defaultTagsJson, ok := vars["aws-native:config:defaultTags"]; ok {
+		var defaultTags map[string]string
+		err := json.Unmarshal([]byte(defaultTagsJson), &defaultTags)
+		if err != nil {
+			return nil, fmt.Errorf("failed to unmarshal 'forbiddenAccountIds' config: %w", err)
+		}
+		p.defaultTags = defaultTags
+	} else {
+		p.defaultTags = make(map[string]string)
```

Comment:
nil maps are usable for lookups, do we need an actual map?

Reply:
Sure, I'll change to a nil assignment. It's kinda an edge case, but when debugging the provider, if we don't allocate a nil here it causes a weird bug when we remove the default tags from the config as the old value will then remain set if we don't explicitly unset it.